docs/src/ref/gfi.md

```diff
@@ -248,9 +266,8 @@ u = \log p(t'; x')/(p(t; x) q(t'; x', t + u)) = \log 0.0294/(0.0784 \cdot 0.1) =
 
 
 ### Regenerate
-```@docs
-regenerate
-```
+The [`regenerate`](@ref) method takes a trace and generates an adjusted trace that is consistent with a change to the arguments to the function, and also generates new values for selected random choices.
```

Comment:
Is it the case that `regenerate` must also resample the non-addressable randomness?

Reply:
Yes. The text here is attempting to be a high-level conceptual introduction of what regenerate does (the precise definition is in the function documentation).